core-bundle/contao/library/Contao/TemplateTrait.php

```diff
@@ -191,6 +192,40 @@ public function addCspHash(string $directive, string $script, string $algorithm
 		$csp->addHash($directive, $script, $algorithm);
 	}
 
+	/**
+	 * Extracts all inline CSS style attributes of a given HTML string and automatically adds CSP hashes for those
+	 * to the current response context.
+	 */
+	public function extractStyleAttributesForCsp(string $html): string
```

Comment:
```suggestion
	public function extractStyleAttributesForCsp(string|null $html): string
```

This method is mainly used for content from database fields which definition is `text NULL`. Thus it either needs to accept `null` - or the caller needs to use `$this->extractStyleAttributesForCsp($this->text ?? '')`.